### DISCUSSION TITLE
Node of richText is missing type

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#1141] [Q&A] [answered]
### Node of richText is missing type

<img width="801" alt="image" src="https://user-images.githubusercontent.com/16400882/190838024-143303ea-2d4f-4005-8eff-3194187a3f1f.png">

<img width="891" alt="image" src="https://user-images.githubusercontent.com/16400882/190838029-773e3d37-6d7c-42be-9c01-19350138fb3c.png">

expected format is:  { "children":[{"text":""}], type: "paragraph" }

**Comment by thompsonsj** [comment, marked as answer]
I believe this is intentional. Slate is [schema-less](https://docs.slatejs.org/#principles), so each implementation of Slate defines its own schema.

In the case of Payload CMS, `paragraph` is not set as a fallback for nodes with no type. This differs from the [Slate demo](https://www.slatejs.org/examples/richtext), where `paragraph` is set on nodes.

In the docs for the rich text field, the serializer example renders the `<p>` HTML element as the default - i.e. if no types are found. See https://github.com/payloadcms/payload/blob/master/docs/fields/rich-text.mdx.

So I am guessing this is what you are expected to do? If the type is undefined, assume it's a paragraph!
